packages/web-components/src/radio-group/radio-group.styles.ts

```diff
@@ -0,0 +1,62 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  colorCompoundBrandForeground1,
+  colorNeutralForeground1,
+  colorNeutralForeground3,
+  colorNeutralStrokeAccessible,
+  fontFamilyBase,
+  fontSizeBase300,
+  fontWeightRegular,
+  lineHeightBase300,
+  spacingHorizontalS,
+  spacingHorizontalXS,
+  spacingVerticalS,
+} from '../theme/design-tokens.js';
+
+/** RadioGroup styles
+ * @public
+ */
+export const styles = css`
+  ${display('flex')}
+
+  :host {
+    --control-border-color: ${colorNeutralStrokeAccessible};
+    --checked-indicator-background-color: ${colorCompoundBrandForeground1};
+    --state-color: ${colorNeutralForeground3};
+    align-items: flex-start;
+    flex-direction: column;
+    row-gap: ${spacingVerticalS};
+  }
+  ::slotted([slot='label']) {
+    color: ${colorNeutralForeground1};
+    padding: ${spacingVerticalS} ${spacingHorizontalS} ${spacingVerticalS} ${spacingHorizontalXS};
+    font: ${fontWeightRegular} ${fontSizeBase300} / ${lineHeightBase300} ${fontFamilyBase};
+    cursor: default;
+  }
+  .positioning-region {
+    display: flex;
+    flex-wrap: wrap;
+  }
+  :host([orientation='vertical']) .positioning-region {
+    flex-direction: column;
+    justify-content: flex-start;
+  }
+  :host([orientation='horizontal']) .positioning-region {
+    flex-direction: row;
+  }
+  :host([orientation='horizontal']:not([stacked])) ::slotted([role='radio']) {
```

Comment:
.Do you need the not selector? :not selectors aren't very performant. Seems like this would be fine:

`:host([orientation='horizontal']) ::slotted([role='radio'])`

Since you are overriding `padding-inline` in `:host([orientation='horizontal'][stacked]) ::slotted([role='radio'])`